app/assets/stylesheets/components/c-footer.scss

```diff
@@ -34,3 +34,25 @@
   }
 }
 
+@media screen and (min-width: 768px) {
+  .c-footer {
+    &-nav {
```

Comment:
Is the block name `c-footer`, `or c-footer-nav` or does the footer have a `__nav` element?

block name `c-footer-nav`  or element?

```
c-footer {
  &__nav {
  }
}
```

Personally, I think `__nav` makes more sense